### DISCUSSION TITLE
A thread to list all duplicate tasks

### DISCUSSION THREAD

**Discussion by Palipoor** [allenai/natural-instructions#701] [General]
Hi
I am creating this thread to list duplicate tasks that we encounter, so we can merge them later on, preferably using the "multiple definitions" @danyaljj suggested.

**Comment by Palipoor** [comment]
I will also use this to ask @danyaljj  
If we use a dataset as a classification task into "0" or "1", does it really make sense to have it as a classification task into "Yes" or "No" or any other pair? I don't think it does.

**Reply by danyaljj** [reply]
I am actually for keeping those separate. It's a measure of whether our models generalize to slightly different output encodings.
